res/css/views/settings/tabs/user/_GeneralUserSettingsTab.pcss

```
@@ -14,39 +14,47 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-.mx_GeneralUserSettingsTab_changePassword .mx_Field {
-    margin-inline-end: var(--SettingsTab_fullWidthField-margin-inline-end);
+:root {
+    --GeneralUserSettingsTab_subheading-margin-block-start: 24px;
```

It would be good to add a comment describing what this variable actually means. 

```suggestion
    // The top margin of subheadings within the `GeneralUserSettingsTab`
    --GeneralUserSettingsTab_subheading-margin-block-start: 24px;
```

maybe?

---

Thanks, I would agree :+1: